modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -403,6 +401,50 @@
     /** Handshake wait message type. */
     public static final short HANDSHAKE_WAIT_MSG_TYPE = -28;
 
+    /** Communication metrics group name. */
+    public static final String COMMUNICATION_METRICS_GROUP_NAME = "communication.tcp";
+
+    /** */
+    public static final String SENT_MESSAGES_METRIC_NAME = "sentMessagesCount";
+
+    /** */
+    public static final String SENT_MESSAGES_METRIC_DESC = "Total number of messages sent by current node";
+
+    /** */
+    public static final String RECEIVED_MESSAGES_METRIC_NAME = "receivedMessagesCount";
+
+    /** */
+    public static final String RECEIVED_MESSAGES_METRIC_DESC = "Total number of messages received by current node";
+
+
+    /** */
+    public static String sentMessagesByTypeMetricName(Short directType) {
+        return "sentMessagesByType." + directType;
```

Comment:
Let's use MetricUtils#metricName instead of this and similar methods.

Reply:
ok